proposals/dao/tip_121c_pt2.ts

```diff
@@ -0,0 +1,113 @@
+  console.log(' pcvStatsBefore.userCirculatingFei      [M]e18 ', Number(pcvStatsBefore.userCirculatingFei) / 1e24);
+  console.log(' pcvStatsBefore.protocolEquity          [M]e18 ', Number(pcvStatsBefore.protocolEquity) / 1e24);
+  const pcvStatsAfter: PcvStats = await contracts.collateralizationOracle.pcvStats();
+  console.log('----------------------------------------------------');
+  console.log(' pcvStatsAfter.protocolControlledValue  [M]e18 ', Number(pcvStatsAfter.protocolControlledValue) / 1e24);
+  console.log(' pcvStatsAfter.userCirculatingFei       [M]e18 ', Number(pcvStatsAfter.userCirculatingFei) / 1e24);
+  console.log(' pcvStatsAfter.protocolEquity           [M]e18 ', Number(pcvStatsAfter.protocolEquity) / 1e24);
+  console.log('----------------------------------------------------');
+  const pcvDiff = pcvStatsAfter.protocolControlledValue.sub(pcvStatsBefore.protocolControlledValue);
+  const cFeiDiff = pcvStatsAfter.userCirculatingFei.sub(pcvStatsBefore.userCirculatingFei);
+  const eqDiff = pcvStatsAfter.protocolEquity.sub(pcvStatsBefore.protocolEquity);
+  console.log(' PCV diff                               [M]e18 ', Number(pcvDiff) / 1e24);
+  console.log(' Circ FEI diff                          [M]e18 ', Number(cFeiDiff) / 1e24);
+  console.log(' Equity diff                            [M]e18 ', Number(eqDiff) / 1e24);
+  console.log('----------------------------------------------------');
+
+  expect(await contracts.collateralizationOracle.isOvercollateralized()).to.be.true;
+
+  // 1. Verify Tribe Redeemer contract deploy params
+  expect(await contracts.tribeRedeemer.redeemBase()).to.equal(REDEEM_BASE);
+  expect(await contracts.tribeRedeemer.redeemedToken()).to.equal(addresses.tribe);
+
+  const expectedTokensReceived = [addresses.steth, addresses.lqty, addresses.fox, addresses.dai];
+  const actualTokensReceived = await contracts.tribeRedeemer.tokensReceivedOnRedeem();
+  expect(actualTokensReceived.length).to.equal(expectedTokensReceived.length);
+  expect(actualTokensReceived).to.contain(actualTokensReceived[0]);
+  expect(actualTokensReceived).to.contain(actualTokensReceived[1]);
+  expect(actualTokensReceived).to.contain(actualTokensReceived[2]);
+  expect(actualTokensReceived).to.contain(actualTokensReceived[3]);
+
+  // 2. Verify Tribe Redeemer has all PCV assets deposited on it
+  // stETH is rebasing, so amount transfered to tribeRedeemer will be greater than current amount on deposit
+  expect(await contracts.steth.balanceOf(addresses.tribeRedeemer)).to.be.bignumber.greaterThan(STETH_DEPOSIT_BALANCE);
+  expect(await contracts.dai.balanceOf(addresses.tribeRedeemer)).to.equal(DAI_HOLDING_DEPOSIT_BALANCE);
+  expect(await contracts.lqty.balanceOf(addresses.tribeRedeemer)).to.equal(DAO_TIMELOCK_LQTY_BALANCE);
+  expect(await contracts.fox.balanceOf(addresses.tribeRedeemer)).to.equal(DAO_TIMELOCK_FOX_BALANCE);
```

Comment:
should add a redeem of 100000 TRIBE to see what the amounts that come out are

Reply:
`test/integration/tests/tribeRedeemer.ts` has 10,000 and 1,000,000 redeem calls and expected amounts out